lib/led/rgb.js

```diff
@@ -293,8 +293,6 @@ class RGB {
     // If it's already off, do nothing so the previous state stays intact
     /* istanbul ignore else */
     if (this.isOn) {
-      state.prev = RGB.colors.reduce((current, color) => (current[color] = state[color], current), {});
```

Comment:
When `off()` is called, there is no need to reassign `state.prev` if the RGB is on.  This also fixes a bug in the the case if `off()` is called while the RGB is pulsing. The RGB values will most likely be assigned lesser values and the light will decrease gradually as `off()` is called after `pulse()` is called.